README.md

```diff
@@ -31,24 +31,26 @@ When a major issue in "beta 3" is found on SOAK, we will follow the steps above
 Only pre-releases that are SOAKed for at least two weeks should be used as stable releases. After the soaking period was successful, follow these steps:
 
 1. fetch latest repo state: `git fetch`
-1. checkout the pre-release tag you want to make a stable release: e.g. `git checkout v1.1.0-rc.3`
-1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`
-1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to your newly created `stable-*` branch (remember to also update revision)
+1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to `master` (remember to also update revision)
 1. apply and push tags
    1. "base" semver based tag: e.g. `git tag v1.1.0 && git push origin v1.1.0`
    1. "consumable" stable tag(s) for supported k8s versions: e.g. `git tag stable-1.16-1.1.0 && git push origin stable-1.16-1.1.0`
+1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`. that way future updates have an easy target.
 1. head to github and update release information for that release: check [releases page](https://github.com/mesosphere/kubeaddons-kommander/releases) for up-to-date example. _Make sure to edit the "base" release_.
-1. add that new stable branch to `mergebot-config.json` on `master` and set its version to the next patch release (usually `.1`)
+1. add that new `stable-*` branch to `mergebot-config.json` on `master` and set its version to the next patch release (usually `.1`)
+1. in order to allow backports to that newely reated minor version, make sure that the charts minor version also is bumped. 
+1. to make it easy for fellow colleagues, create a new directory (e.g. `1.2.0` for people to work in) 
```

Comment:
I included this step in a suggested step above, so we can remove. Also - should this new dir be the next patch version instead of minor version?